src/create_py_project/types.py

```
@@ -0,0 +1,64 @@
+    devcontainer: bool = True
+    open_source_license: Literal[
+        "MIT license",
+        "BSD license",
+        "ISC license",
+        "Apache Software License 2.0",
+        "GNU General Public License v3",
+        "Not open source",
+    ] = "MIT license"
+
+    @classmethod
+    def create(cls, *, project_name: str, **kwargs: object) -> "ProjectConfig":
```

Ruff (0.15.15)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)